adoptadog-javascript.js

```diff
@@ -0,0 +1,6 @@
+function addToCart() {
+    let total = 123.45;
+    for (i=1; i<=total; i++)
+    alert(total * i); 
+}
+
```

Comment:
You'll want to move the total out of this function, and start it at 0;

Then when you call this function all you have to do is add in some amount to that total. Even better, passing in some amount from the HTML

something like

```
let total = 0;

function addToCart(dogCost) {
    total += dogCost;
    alert(total);
}
```